diff

Fix http://bugs.activestate.com/show_bug.cgi?id=38281
"AutoComplete only shows on Primary Monitor"

See http://bugs.activestate.com/show_bug.cgi?id=38281#c27
for an explanation of the issue.

Index: win32\PlatWin.cxx
===================================================================
--- win32/PlatWin.cxx	(revision 22266)
+++ win32/PlatWin.cxx	(working copy)
@@ -12,8 +12,6 @@
 #include <vector>
 #include <map>
 
-#undef _WIN32_WINNT
-#define _WIN32_WINNT  0x0500
 #include <windows.h>
 #include <commctrl.h>
 #include <richedit.h>

